src/Controller/SpecialBenefit/Edit.php

```diff
@@ -14,7 +14,7 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\RouterInterface;
-use Twig_Environment As Twig;
+use Twig_Environment as Twig;
```

Comment:
```suggestion
use Twig\Environment as Twig;
```

Reply:
Fixed here and in others files where Twig_Environment was used